### PR TITLE
devapp: update to v1_beta11

### LINKED ISSUE #141: 'multer': middleware not shipped with dev-app standalone tar.gz
@brunobottazzini 
multer node module does not come together with soletta-dev-app_standalone_v1_beta10.tar.gz and cause dev-app-server not able to start.
User need to manually run npm to install it.
Can this be included in the package or recipe?

### PR DESCRIPTION
**Pull request by yongli3**
To include multer module, to fix https://github.com/solettaproject/soletta-dev-app/issues/141

Signed-off-by: Yong Li yong.b.li@intel.com

### CODE REVIEW

**Comment by yongli3** [comment]
@anselmolsm @fulong82 @chongyic please help to review

File: recipes-soletta/dev-app/soletta-dev-app_git.bb [hunk 1]
```diff
@@ -2,12 +2,12 @@ DESCRIPTION = "Soletta Development Application"
...
-PV = "1_beta10"
+PV = "1_beta11"
 
 LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=93888867ace35ffec2c845ea90b2e16b"
-SRC_URI[md5sum] = "00f1c0b970cc8d682c7a20c960f4b8e7"
-SRC_URI[sha256sum] = "98ab16895a65692221035998a73d830b04a6835c672c9d64cac2010431519f8e"
-SRC_URI = "https://github.com/solettaproject/soletta-dev-app/releases/download/v${PV}/soletta-dev-app_standalone_v${PV}.tar.gz;protocol=archive \
+SRC_URI[md5sum] = "9c4dfd30cad368c3d57542eff288d33d"
+SRC_URI[sha256sum] = "8bfd68dd20fd8bb77c9080600f3ae36e13ea537f31ebf335eb83a6e0dec6c80b"
+SRC_URI = "https://github.com/solettaproject/soletta-dev-app/releases/download/v${PV}/soletta-dev-app_standalone_v${PV}-2.tar.gz;protocol=archive \
```
Comment (fulong82) [inline recipes-soletta/dev-app/soletta-dev-app_git.bb]: Isn't it more clean if the "-2" put together at PV = "1_beta11-2" ?

Comment (yongli3) [reply recipes-soletta/dev-app/soletta-dev-app_git.bb]: The URL is https://github.com/solettaproject/soletta-dev-app/releases/download/v1_beta11/soletta-dev-app_standalone_v1_beta11-2.tar.gz

There are two ${PV}: /v1_beta11/ and v1_beta11-2.tar.gz, it seems that the URL needs to be changed?

**Comment by peijiajames** [comment]
I suppose v1_beta11-2 is the temporary name and we will have formal one for official release.

**Comment by anselmolsm** [comment]
@peijiajames is right, the "-2" is temporary. dev-app itself is the same in both v1_beta11 and v1_betta11-2, the difference was regarding the number of modules shipped inside node_modules.

**Comment by anselmolsm** [comment]
Done. One v1_betta11 tar.gz to rule them all.

File: recipes-soletta/dev-app/soletta-dev-app_git.bb (hunk 1 above)
Comment (anselmolsm) [reply recipes-soletta/dev-app/soletta-dev-app_git.bb]: Now there is a single package for v1_beta11. The "-2" one was for tests, to validade the release generation script was working fine.

File: recipes-soletta/dev-app/soletta-dev-app_git.bb [hunk 2]
```diff
@@ -2,12 +2,12 @@ DESCRIPTION = "Soletta Development Application"
...
 LICENSE = "Apache-2.0"
 
-PV = "1_beta10"
+PV = "1_beta11"
 
 LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=93888867ace35ffec2c845ea90b2e16b"
-SRC_URI[md5sum] = "00f1c0b970cc8d682c7a20c960f4b8e7"
-SRC_URI[sha256sum] = "98ab16895a65692221035998a73d830b04a6835c672c9d64cac2010431519f8e"
-SRC_URI = "https://github.com/solettaproject/soletta-dev-app/releases/download/v${PV}/soletta-dev-app_standalone_v${PV}.tar.gz;protocol=archive \
+SRC_URI[md5sum] = "9c4dfd30cad368c3d57542eff288d33d"
```
Comment (anselmolsm) [inline recipes-soletta/dev-app/soletta-dev-app_git.bb:8]: Why both md5sum and sha256sum ?

Comment (yongli3) [reply recipes-soletta/dev-app/soletta-dev-app_git.bb:8]: @anselmolsm Do you mean we can remove md5sum or sha256sum line? The original bb file includes both of them, and other .bb files are using both of them too

Comment (anselmolsm) [reply recipes-soletta/dev-app/soletta-dev-app_git.bb:8]: Yeah, I saw the other one is also using both as well, I am wondering why.
But it's a minor thing.

**Comment by okartau** [comment]
test this please

File: recipes-soletta/dev-app/soletta-dev-app_git.bb (hunk 1 above)
Comment (yongli3) [reply recipes-soletta/dev-app/soletta-dev-app_git.bb]: Thanks @anselmolsm I updated the PR to remove the "-2"

**Comment by fulong82** [comment]
@chongyic is this commit work for you? Thx.

**Comment by chongyic** [comment]
Hi @okartau @fulong82 @yongli3 @anselmolsm @peijiajames 
The fix works like a charm :)

**Comment by peijiajames** [comment]
@chongyic thanks for the verification. @anselmolsm @ceolin could you help to merge this?

**Comment by anselmolsm** [comment]
@peijiajames done.

**Pull request closed without merging**